extensions/amp-story/1.0/amp-story-system-layer.js

```diff
@@ -361,6 +443,11 @@ export class SystemLayer {
    */
   onUnmuteAudioClick_() {
```

Comment:
This method looks duplicated

Reply:
It is pretty much the same as onMuteAudioClick(), except on this one, the first line is calling to toggle the muted attribute to false, and the other it toggles it to true. They both are the same besides that because they have to show the message regardless and disappear, with the only difference being the type of message that is displayed. If this is too much code repetitiveness, I can refactor consolidate the method and then pass in a boolean variable for the corresponding mute/unmute, so let me know if you think that sounds good!

Reply:
+1 to the refactor suggested here